posthog/tasks/test/test_process_event.py

```diff
@@ -63,7 +90,7 @@ def test_alias(self) -> None:
                 'token': self.team.api_token,
                 'alias': 'old_distinct_id'
             },
-        }, self.team.pk, now().isoformat())
+        }, self.team.pk, now().isoformat(), now().isoformat())
```

Comment:
Shouldn't you be able to pass through None to all of these?

Reply:
test added for that